build_elexis/assets/build_inside_container

```diff
@@ -7,19 +7,27 @@ build_root=/home/elexis
 cd $build_root
 if [ -d "$build_root/elexis-3-core" ]
 then
-  echo "Don't clone elexis-3-core"
+  echo "pull elexis-3-core"
+  cd elexis-3-core
+  git pull
+  cd ..
 else
+  echo clone elexis-3-core
   git clone https://github.com/elexis/elexis-3-core
 fi
 cd elexis-3-core
 # Add -Dmaven.test.skip=true if you have problems running the tests, eg. no X display
-echo mvn clean install  $MAVEN_OPTS_ELEXIS_3_CORE
-mvn clean install  $MAVEN_OPTS_ELEXIS_3_CORE
+echo mvn clean install  $MAVEN_OPTS_ELEXIS_3_CORE -Dmaven.test.skip=true
+mvn clean install  $MAVEN_OPTS_ELEXIS_3_CORE -Dmaven.test.skip=true
 cd $build_root
```

Comment:
No. It is bad practive to skip running unit tests. And as I want to use this dockerfiles for my Jenkins I am okay that users may disable the unit tests, but it must be possible to run them, too.

Therefore you might set the default options to skip the test, but not hard-wire them.

If you fix this, I will be happy to merge your changes.